pkg/scheduler/framework/plugins/noderesources/fit.go

```diff
@@ -310,7 +310,8 @@ func fitsRequest(podRequest *preFilterState, nodeInfo *framework.NodeInfo, ignor
 				continue
 			}
 		}
-		if rQuant > (nodeInfo.Allocatable.ScalarResources[rName] - nodeInfo.Requested.ScalarResources[rName]) {
+		available := nodeInfo.Allocatable.ScalarResources[rName] - nodeInfo.Requested.ScalarResources[rName]
+		if rQuant > available && rQuant != 0 {
```

Comment:
`available` cannot be a negative number. So if `rQuant` > `available`, `rQuant`  must be larger than `0`.
So `&& rQuant != 0` is useless.